types/undertaker/index.d.ts

```diff
@@ -46,20 +52,21 @@ declare class Undertaker extends EventEmitter {
 
     /**
      * Returns the registered function.
-     * @param taskName - Task name.
```

Comment:
Why you removed js-docs comments?

Reply:
@GiedriusGrabauskas  because I was getting an error saying that this isn't necessary in typescript as its inferred from the implementation.